### ISSUE TITLE
Distinguish the difference/similarity/future between ipwb and pywb-ipfs

### ISSUE THREAD

**Issue by machawk1** [oduwsdl/ipwb#9]
https://github.com/ikreymer/pywb-ipfs

**Comment by ikreymer** [comment]
Yes, I think this merits some discussion.. As I've mentioned before, the main issue is that there is not a standard way to write WARCs into IPFS.. pywb-ipfs tried one approach, ipwb tried a different, Juan suggested a different approach.. I don't have a particular preference, nor time to resolve this.. but I think it is really important to standardize here, especially if this is continuing beyond proof-of-concept stage into actual production.
I have been holding off on integrating ipfs support for this reason.

**Comment by ibnesayeed** [comment]
That translates to me that we really need a video conference some day.

**Comment by machawk1** [comment]
@ikreymer I would be interested in an asynchronous discussion on these approaches. If nothing else, listing out the pros and cons of each of ipwb, pywb-ipfs, and @jbenet's approach would be in the right step for standardization of WARCs into IPFS.

**Comment by ikreymer** [comment]
Yes, I definitely agree re: video conference. Hopefully @jbenet can join in as well, or someone else from IPFS. I would really like to decide on a standard way to store WARC files, as well as CDXJs indexes so that different tools that write/read web archive data to IPFS can stay compatible.

I am trying hard to build distinct components that can be interoperable with other tools and could be used independently, to avoid duplicating work. For example, I would like to add IPFS support to Webrecorder at some point after we have agreed on standard way to do it. Separately, adding WARC upload into Webrecorder is also planned, which could then provide WARC upload into IPFS. Making Webrecorder a standalone application is another goal. It seems like there is a lot of overlap with what you are trying to do with ipwb (referring to #26), so it would be great to figure out a way that we can work together on some of these tasks..
